docs/design/expressions/README.md

```diff
@@ -69,6 +69,9 @@ graph BT
                   x->(...)"]
     click memberAccess "https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/expressions/member_access.md"
```

Comment:
Hm, is there a way we can give different links here for member access vs call vs indexing?

Reply:
Done.